tap/passive_tapper.go

```diff
@@ -96,6 +101,20 @@ func StartPassiveTapper(opts *TapOpts, outputItems chan *api.OutputChannelItem,
 	go startPassiveTapper(opts, outputItems)
 }
 
+func UpdateTapTargets(newTapTargets []v1.Pod) {
+	tapTargets = newTapTargets
+	initializePacketSources()
+	printNewTapTargets()
+}
+
+func printNewTapTargets() {
+	printStr := ""
+	for _, tapTarget := range tapTargets {
+		printStr += tapTarget.Status.PodIP + " "
+	}
+	logger.Log.Infof("Now tapping: %s", printStr)
```

Comment:
should we print a pod name as well, as this is what user see, and will open github issues about..

Reply:
very good point